.github/workflows/check_merge_reqs.yml

```diff
@@ -20,8 +24,7 @@ jobs:
         run: echo "label-checks-pass=true" >> $GITHUB_ENV
 
       - name: Final Checks
-        if: >
-          env.label-checks-pass == false
+        if: env.label-checks-pass == false
```

Comment:
Is unset the same as "false" here? If not, might want to go with `!= true` instead (or some other defensive pattern.)

Reply:
switched to `${{ !env.label-checks-pass }}` in c564831b